tensorboard/data/server/BUILD

```diff
@@ -85,6 +86,7 @@ genrule(
     srcs = [
```

Comment:
To double check, humans never need to manually run this `gen_protos` genrule; `update_protos` is the only target in this folder that we ever need to manually `bazel run`?

Reply:
Right. I think that’s true in general—it should never be necessary to
run a genrule, since all a genrule does is build its outputs, and those
will be built automatically if they’re transitively required.